content/faculty/honours_board.md

```diff
@@ -135,3 +143,129 @@ submenu_index: 4
     </table>
   </div>
 </div>
+
+
+<div class="tab-content board-2016" style="display: none">
```

Comment:
You shouldn't use inline CSS. Find out if you can use some existing class for this. Write one if there is no such a class.